src/docserv/functions.py

```diff
@@ -32,6 +33,49 @@ def mail(text, subject, to):
     p.communicate(msg.as_string())
 
 
+def get_all_file_paths(directory, zip_formats):
```

Comment:
This function looks weird, could't find a prettier way. Help would be welcome :)

Reply:
What do you mean by "weird"? The name? Or the way how it is implemented?

Reply:
The way it is implemented